docpages/03_installing.md

```diff
@@ -9,4 +9,4 @@ There are many ways to install D++, either from a package manager, or from sourc
 * \subpage install-windows-vs-zip
 * \subpage install-windows-clion-vcpkg
 * \subpage install-xmake
-* \subpage install-from-source
+* \subpage install-from-source
```

Comment:
Why has this flagged as a changed?

Reply:
New line at the end of file was removed. Github marks it on the line before
The new line *should* be there by UNIX convention

Reply:
As already discussed on Discord, never had an issue with this in all my years on Linux, but if they're wanted, happy to put these back. Hasn't even been me doing any explicit decision here, just the way my editor has been set since ages past.